src/wallet/wallet2.cpp

```diff
@@ -539,10 +541,10 @@ boost::optional<tools::password_container> get_password(const boost::program_opt
     return tools::password_container{command_line::get_arg(vm, opts.password)};
   }
 
-  if (command_line::has_arg(vm, opts.password_file))
+  if (!password_file.empty())
```

Comment:
There is is_arg_defaulted, which does what I think you want here.

Reply:
Thanks for pointing that out! I'll use that instead.